Attestation/README.md

```diff
@@ -780,54 +780,92 @@ for secret transport.  This list is almost certainly not exhaustive.
+[name](/Intro/README.md#Cryptographic-Object-Naming) of the
+authorization key, along with a small secret to transport.  The caller
+of `TPM2_ActivateCredential()` must then provide the handles for those
+two key objects and the outputs of `TPM2_MakeCredential()` in order to
+extract the small secret.  Typically the small secret is an AES key for
+encrypting larger secrets.
+
+So if we can store the outputs of `TPM2_MakeCredential()` long-term so
+that the client can activate over multiple reboots, then we have a way
+to deliver secrets to the client.
+
+We'll discuss two ways to do this:
+
+ - use a universally well-known key -- let's call it the `WK`
+
+   Since the `WK`'s private area is not used for any cryptography in
+   `TPM2_MakeCredential()`/`TPM2_MakeCredential()`, it can be a key that
+   everyone knows.
+
+   Note that the `WK`'s public area can name arbitrary an auth policy,
+   and `TPM2_MakeCredential()` will enforce it.
+
+   E.g., the `WK` could be the all-zeros AES key.  Its policy could be
+   whatever is appropriate for the organization.  For example, the
+   policy could require that some non-resettable application PCR have
+   the value zero so that extending it can disable use of
+   `TPM2_MakeCredential()` post-boot.
+
+ - use a long-term `AK` -- let's call it the `LTAK`
+
+   I.e., an `AK` that lacks the `stClear` attribute, and preferably
+   created deterministically with
+   [`TPM2_CreatePrimary()`](/TPM-Commands/TPM2_CreatePrimary.md) or
+   [`TPM2_CreateLoaded()`](/TPM-Commands/TPM2_CreateLoaded.md).
+
+   Note that the `AK`'s public area can name arbitrary an auth policy,
+   and `TPM2_MakeCredential()` will enforce it.
+
+The best option here is to use a `WK` because using an `LTAK` would
+require recording its public key in the device's enrolled attestation
+state, which would complicate enrollment, whereas the `WK`, being
+well-known and the same for all cases, would not need to be recorded in
+server-side attestation state.
+
+> One might like to use the `EK` as the `activateHandle`.  Sadly, this
+> is not possible.
+> While `TPM2_MakeCredential(EKpub, EKname, input)` works,
+> `TPM2_ActivateCredential(EK, EK, credentialBlob, secret)` does not
+>  and cannot.
+>
+> The reason for this is that `TPM2_ActivateCredential()` requires
+> `ADMIN` role for the `activateHandle`, and since the `EK` has
+> `adminWithPolicy` attribute set and its policy doesn't have the
+> `TPM_CC_ACTIVATECREDENTIAL` command permitted, the call must fail.
+>
+> Credit for the `WK` idea goes to [Erik > Larsson](https://developers.tpm.dev/chats/new?user_id=4336638).
+
+Normally during attestation we want to use an `AK` with `stClear` set so
+that each boot forces the client to use a new one.  However, for sending
+secrets to the client via `TPM2_MakeCredential()` /
+`TPM2_ActivateCredential()` we really need need the `activateHandle`
+object to not have `stClear` set.
+
+For this approach then, the best solution is to use a `WK`.
```

Comment:
A diagram of interactions , or sequence would be helpful to visualize this. I could try to make one, unless you prefer to do it.

Reply:
Sure, I should probably update the planuml diagrams I already have.